src/main/java/io/appium/java_client/touch/offset/PointOption.java

```diff
@@ -36,14 +32,14 @@ public static PointOption point(int xOffset, int yOffset) {
      * @return self-reference
      */
     public T withCoordinates(int xOffset, int yOffset) {
-        checkArgument(xOffset >= 0, format(ERROR_MESSAGE_TEMPLATE, "X"));
```

Comment:
Is it temporary change for backward compatibility with older servers versions?
I think it is not necessary to remove it. Just comment it, maybe with the TODO mark.
And before the publishing of the 6.0.0 we can remove `//`

Reply:
There might be a situation when it is necessary to provide negative absolute coordinates. For example, if the device has two screens and the second screen is located on the left side of the main one. That is why I'd consider this restriction as not very useful.

Reply:
Also, it is possible that one wants to drag an element to an off-screen area and negative coordinates might also have sense for such case.

Reply:
I agree with @mykola-mokhnach. I tried using java-client 6.0.0-BETA1, but I was needed to rollback due to the new limitation: we have cases when actions start at the screen, but end out off screen.